### PR TITLE
dupseek 1.3 (new formula)

### PR DESCRIPTION
**Pull request by bf4**
Interactive program to find and remove duplicate files.

Why?

It's old, yeah, but it's been stable and effective for years.
It works really well and I like the command-line options
better than duff, fdupes, rdfind, fdup, etc.

Fun fact: My last [PR here](https://github.com/Homebrew/homebrew/pulls?q=is%3Apr+author%3Abf4+is%3Aclosed) was [writerperfect](https://github.com/Homebrew/homebrew/pull/4917) in 2011. Things sure have changed.

### CODE REVIEW

File: Library/Formula/dupseek.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  # Dupseek (and destroy) is Copyright Antonio Bellezza 2003-2005.
+  # It is released under the GPL v2.
+  #
+  # This program is free software; you can redistribute it and/or modify
+  # it under the terms of version 2 of the GNU General Public License
+  # as published by the Free Software Foundation;
+  # This program is distributed in the hope that it will be useful,
+  # but WITHOUT ANY WARRANTY; without even the implied warranty of
+  # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  # GNU General Public License for more details.
```
Comment (mikemcquaid) [inline Library/Formula/dupseek.rb]: Remove all this license text.

File: Library/Formula/dupseek.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+  def install
+    bin.install "dupseek"
+    doc.install %w[changelog.txt doc.txt copyright credits.txt]
+  end
+
+  test do
+    mkdir "folder"
+    touch "folder/file1"
+    assert_equal "", pipe_output("#{bin}/dupseek -b report -f de folder", nil, 0).chomp
```
Comment (mikemcquaid) [inline Library/Formula/dupseek.rb]: Looks like you aren't piping anything so can use `shell_output`

**Comment by bf4** [comment]
@mikemcquaid Thanks for the quick review! Updated per comments.  Is there documentation of the utilities I can use in test?  I just looked through existing formula for what looked to do something useful

**Comment by mikemcquaid** [comment]
> Is there documentation of the utilities I can use in test? I just looked through existing formula for what looked to do something useful

The documentation is lacking a bit; that's the best approach.

**Comment by apjanke** [comment]
Merged. Thank you for your contribution to Homebrew!

**Comment by bf4** [comment]
@apjanke Thanks. That was fun!

**Pull request closed without merging**